src/app/wallet/wallet/send/send.component.html

```diff
@@ -16,7 +16,7 @@
       </mat-tab>
     </mat-tab-group>
 
-    <div class="buttons">
+    <div class="buttons" *ngIf="send.input === 'anon' || type === 'sendPayment'">
```

Comment:
Prefer to use the `TxType` types if `send.input` is of the type `TxType`. 
For example: replace with something like: 
`send.input === TxType.ANON || type === 'sendPayment'`